app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1376,6 +1376,20 @@ def _validate_sticky_enrollment(self, data):
 
         return data
 
+    def _validate_rollout_version_support(self, data):
+        if not self.instance or not self.instance.is_rollout:
+            return data
+
+        for support in NimbusExperiment.ROLLOUT_SUPPORT:
+            if NimbusExperiment.Version.parse(
+                self.instance.firefox_min_version
+            ) < NimbusExperiment.Version.parse(support.firefox_min_version):
```

Comment:
Wait this is a bit wonky, it's not matching by application?  This is matching all supported version for all applications against the experiment.  I think what you want here is the constant being a dict that maps application to minimum supported version and then look up the version from the experiments application?

Reply:
ahaha yeah that's totally my bad. 👍